samples/17.b.clear-after-idle/package-lock.json

```diff
@@ -3381,6 +3381,7 @@
             "minipass": {
```

Comment:
Why is the sample package-lock being change?

Reply:
I think I did npm audit a while back and this popped up.